src/main/resources/google_checks.xml

```diff
@@ -55,7 +55,7 @@
       <property name="format"
                value="\\u00(09|0(a|A)|0(c|C)|0(d|D)|22|27|5(C|c))|\\(0(10|11|12|14|15|42|47)|134)"/>
       <property name="message"
-               value="Consider using special escape sequence instead of octal value or Unicode escaped value."/>
+               value="Consider using special escape sequence instead of octal value or Unicode escaped value."/> <!-- SUPPRESS CHECKSTYLE Can't split long messages between lines. -->
```

Comment:
There should be no changes in this file.

Reply:
The only other option I see is mentioning the contents of the line then? Something like this:
```xml
  <module name="SuppressWithNearbyTextFilter">
    <property name="nearbyTextFormat"
              value="Consider using special escape sequence instead of octal.*Unicode"/>
    <property name="idFormat" value="lineLengthXml"/>
  </module>
```

Reply:
Done though.

Reply:
Feels like the original problem(of maintaining 2 files for this suppression) still persists. If this line ever changes, filter config should also be changed. I suppose it is a lot less likely than say moving the line around, but still.

Reply:
Yup, as long as you are okay with it. Done